src/Components/Description/Descrtption.tsx

```diff
@@ -0,0 +1,131 @@
+import "./Description.css";
+
+export default function Description() {
+  return (
+    <>
+      <section className="description-section">
+        <div className="description-section-navbar">
+          <ul>
+            <li className="description-section-navbar-active">Description</li>
+            <li>Additional Info</li>
+            <li>Reviews</li>
+            <li>Video</li>
+          </ul>
+        </div>
+
+        <div className="description-section-content">
+          <p className="description-section-content-title">Varius tempor.</p>
+          <p className="description-section-content-subdescription">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ac
+            quam dolor. In dignissim lectus sed nisl tempor, ac porttitor libero
+            consectetur. Pellentesque diam dolor, tincidunt nec ante congue,
+            tincidunt facilisis tortor. Mauris vitae massa molestie, sagittis
+            ligula vel, egestas massa. Phasellus quis sodales augue. Donec nec
+            ultricies diam. Integer feugiat odio ut dictum viverra. Donec
+            vehicula nisi placerat cursus mollis. Nunc aliquam tempor justo, ut
+            sagittis nisi. Mauris ullamcorper quis nisl sed dictum. Maecenas
+            quam risus, congue quis accumsan at, imperdiet sed lectus. Aliquam
+            in est purus
+          </p>
+          <p className="description-section-content-title">More Details.</p>
+          <div className="description-section-content-list">
+            <ul>
+              <li>
+                <svg
+                  width="16"
+                  height="16"
+                  viewBox="0 0 16 16"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M2 8.5L5.5 12L14 3.5"
+                    stroke="#8A8FB9"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                  />
+                </svg>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras ac
+                quam dolor. In dignissim lectus sed nisl tempor, ac porttitor
+                libero consectetur.
+              </li>
+              <li>
+                <svg
```

Comment:
Create an icons folder, place all the SVGs there, and use them by importing them where needed.

Reply:
Okay

Reply:
@mthomasgrid This is okay, but we can create a separate component for the SVG, move all the SVG code into that component, and then import the SVG component wherever needed, attaching the code for reference.
```
import React, { SVGProps } from 'react';

export const SortListIcon: React.FC<SVGProps<SVGSVGElement>> = (props) => (
  <svg
    className="fill-ford-fill-interactive  group-hover/sortbutton:fill-skyview-1200"
    fill="none"
    height="24"
    role="img"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
    data-testid="sort-list-icon"
  >
    <title>SortListIcon</title>
    <path
      className="fill-ford-fill-interactive  group-hover/sortbutton:fill-skyview-1200"
      clipRule="evenodd"
      d="M4.8 7.2V12H6V7.2H7.8L5.4 3L3 7.2H4.8ZM8.52 6L9.18 7.2H19.2V6H8.52ZM7.2 10.8H16.8V12H7.2V10.8ZM14.82 15.6H4.8V16.8H15.48L14.82 15.6ZM19.2 15.6V10.8H18V15.6H16.2L18.6 19.8L21 15.6H19.2Z"
      fillRule="evenodd"
    />
  </svg>
);

```